### ISSUE TITLE
Containers need to see each other

### ISSUE THREAD

**Issue by wkoertgen** [wkoertgen/train.docker.dataguard#3]
When it comes to the creation of a standby database - with a duplicate from active database, at least - the containers want to see each other mutually. The scripts have been tested with 2 physical hosts in the same network. I guess we need to work with ambassador container or Docker Compose ...

**Comment by mkoertgen** [comment]
Exactly, `docker --link` should be the way to go. As you said this can be automated with [docker-compose](https://docs.docker.com/compose/) amongst others. 

This has the major advantage that the containers will be able to use hostnames (instead of ips) for communication.
